### ISSUE TITLE
Tweaking some commands & options related to temporary administrators

### ISSUE THREAD

**Issue by BeanerLVP** [LVPlayground/playground#598]
I apologize for bringing this up again; you guys probably won't like this, but I feel that our recent changes require a few more tweaks. We often grant admin rights to plenty of inexperienced players nowadays, so let's take some precautions. This is obviously not a critical issue, but I'd just like to post it for future reference.

**For temporary administrators:**

1. **/property** displays only one option: /property search [name/id]. Which is great, since we decided to disable the rest. Similarly, I think it's best to remove the **create** and **modify** options from the **/house** command's usage info, since they don't work for temporary admins anyway. Why make them seem broken?

2. Disable **/lvp access** altogether. There is no need for temp or inexperienced admins to change access levels for /lvp access -> **/account** & /lvp access -> **/playersettings**, intentionally or not. All admins already have /fly, /boost, /jetpack, /fancy and /rampcar available to them, so it should suffice for fun purposes.

3. Under **/p info** or **/p,** remove **deathmessage**, **giveadmin**, and **takeadmin**, as it doesn't work for them.

4. Two scenarios for /v save & v delete:

- When /v [ID] save or /v [ID] delete are used, there is no message displayed. The commands appear to be broken (but are working as intended, since we've disabled them).
- When /v save or /v delete are used without specifying IDs, the error displayed is:
`Error: Sorry, but you're already driving a vehicle!` - which doesn't make sense.
TL;DR: Remove **save** and **delete** from the usage of /v.

5. Under /my stats and /getstats, change the user level from Administrator to Temporary administrator, or their regular level (Regular Player / VIP). This is probably why names appear red under !players, as discussed in issue #533.

6. Subsequently, change the level to "Temporary administrator" under[ /admins](https://i.imgur.com/vhNL9Xr.png), too. Why not display accurate information? The "(temp'd by ABC)" portion works as intended -- only crew members can view it, so that can remain unchanged.

Everything else seems perfectly fine. Thank you.

**Comment by RussellLVP** [comment]
Hi @BeanerLVP,

I have trouble understanding why we should spend time on this. Temporary administrators are, per definition, trusted. These are not technical issues, these are policy issues. A command like `/guide` could help here.

Temporary administrators are something _we_ decide on. From a player's perspective, there is absolutely no difference. We expect players to follow the instructions of temporary administrators. We expect them to listen, be considered, respectful and treat them like any other admin. When they get banned, they got banned by the temporary administrator, not by whoever decided to give them rights.

If anything, I'm inclined to remove this notion altogether from `/admins` in-game too.

To your points:

  1. This is tiny, done.
  2. I see no reason to restrict this. They can ban players, why shouldn't they be able to give them a ramp car? They need to be able to have fun too.
  3. This is tiny, done.
  4. This is tiny, done.
  5. I see no reason to change this. For all intents and purposes they are administrators.
  6. I see no reason to change this. For all intents and purposes they are administrators.

Thanks,
Russell

**Comment by BeanerLVP** [comment]
Thanks for making the changes @RussellLVP 

The intention behind point 2 is to prevent any mistakes/changes from being made to /account and /playersettings via /lvp access. 

/fly, /boost, /jetpack, /fancy and /rampcar are enabled by default for administrators, so it's not like they need to change any access levels to use them. That was never an objection.

Regarding 5 & 6: Agreed.

**Comment by RussellLVP** [comment]
Until we have evidence that this is a concern and a problem in practice, let's not change this. These commands are powerful, and mistakes will be made, regardless of level of experience. That also goes for commands like `/lvp settings`, which are available to all Management members, but really only understood by one or two people.

Regarding my `/lvp access` comment: yes, they can access the commands themselves, but the command also enables them to grant others access to them. They should be encouraged to do that when it contributes to the fun people are having. In some cases they should be **expected** to do so: it wouldn't be fair to not grant players access to rampcar if they themselves are playing with the feature in a public area.
